demos/TicTacToe/game_resources/Classes/main_menu_scene.cc

```diff
@@ -781,244 +831,248 @@ void MainMenuScene::UpdateUserRecord() {
   user_wins_ = future_wins.result()->value().int64_value();
   user_loses_ = future_loses.result()->value().int64_value();
   user_ties_ = future_ties.result()->value().int64_value();
-  user_record_label_->setString("W:" + to_string(user_wins_) +
-                                " L:" + to_string(user_loses_) +
-                                " T:" + to_string(user_ties_));
 }
 
-// Creates a rectangle of Size size and centered on the origin.
-// Optional parameters: background_color, border_color, border_thickness.
-DrawNode* MainMenuScene::CreateRectangle(Size size, Vec2 origin,
-                                         Color4F background_color,
-                                         Color4F border_color,
-                                         int border_thickness) {
-  // Creates the corners of the rectangle.
-  Vec2 corners[4];
-  corners[0] = Vec2(origin.x - (size.width / 2), origin.y - (size.height / 2));
-  corners[1] = Vec2(origin.x - (size.width / 2), origin.y + (size.height / 2));
-  corners[2] = Vec2(origin.x + (size.width / 2), origin.y + (size.height / 2));
-  corners[3] = Vec2(origin.x + (size.width / 2), origin.y - (size.height / 2));
-
-  // Draws the rectangle.
-  DrawNode* rectangle = DrawNode::create();
-  rectangle->drawPolygon(corners, /*number_of_points=*/4, background_color,
-                         border_thickness, border_color);
-  return rectangle;
-}
+// Sets the user record variables to reflect the user record local variables.
+void MainMenuScene::GetUserRecord() {
+  void MainMenuScene::SetUserRecord() {
```

Comment:
Is this a function defined in a function?

Reply:
This seems wrong.  I think you should double check if you meant to do this.

Reply:
Yup it is wrong. Accidentally copied an extra line when I was updating comments.